doc/client_lua_api.txt

```diff
@@ -607,6 +607,11 @@ Helper functions
     * e.g. `string:split("a,b", ",") == {"a","b"}`
 * `string:trim()`
     * e.g. `string.trim("\n \t\tfoo bar\t ") == "foo bar"`
+* `string:encode_uri()`: Equivalent to the JavaScript one, see [`encodeURI`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI)
+	* Allows reserved & unescaped characters
```

Comment:
I wonder if you should use the examples from MDN to make this clear

Reply:
Is that really necessary? We have zlib compression, base64 etc. without any examples as it's clearly specified externally already.